### PR TITLE
misc tweaks

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: presentation/scripts/lab1-1.sh [hunk 1]
```diff
@@ -19,12 +19,14 @@ CID=$(cat out)  # Save the container ID
...
 
 # Push the image to the registry so Kubernetes wil find it
 comment "Push the image to the IBM Cloud registry"
 doit docker push ${IMAGE_NAME}:v1
 
-comment "Verify its there"
+comment "Verify it's there"
 doit bx cr images
+
+comment --pauseafter "*** End of "$(basename $0)
```
Comment (MHBauer) [inline presentation/scripts/lab1-1.sh]: I like these a lot

File: presentation/scripts/lab1-3.sh [hunk 2]
```diff
@@ -17,12 +17,16 @@ WORKER_IP=$(cat out | grep publicIP | sed "s/.*\"\([0-9].*\)\".*/\1/g" )
...
-comment --nolf "Notice the 'publicIP' field"
+comment --pauseafter "Notice the 'publicIP' field"
+
 comment "Get the nodePort of the service"
 doit kubectl get svc hello-world -ojson
 
 SERVICE_PORT=$(cat out | grep nodePort | sed "s/.*: *\([0-9]*\).*/\1/g")
 
-comment --nolf "Notice the 'nodePort' field"
+comment --pauseafter "Notice the 'nodePort' field"
```
Comment (MHBauer) [inline presentation/scripts/lab1-3.sh]: Did not notice this until now. We are intentionally printing out the json.

Comment (duglin) [reply presentation/scripts/lab1-3.sh]: yea, is that a concern? I wanted to show the full output and then draw peole's attention to the nodePort.

Comment (MHBauer) [reply presentation/scripts/lab1-3.sh]: no issue, was one of the things I wanted to ask about.

**Pull request merged**